src/status_im/ui/screens/home/views.cljs

```diff
@@ -188,6 +189,8 @@
                                         [home-tooltip-view]
                                         [react/view {:height 68}])}])))
 
+(def ENS_BANNER_EXPIRE_TIMESTAMP 1665010800)
```

Comment:
we don't generally use upper case for const I believe

Reply:
yes, sorry brain still not recovered from `DISABLE_HERMES` mindset :D